modules/installation-gcp-permissions.adoc

```diff
@@ -13,317 +13,19 @@ endif::[]
 ifeval::["{context}" == "installing-gcp-user-infra-vpc"]
 :template:
 endif::[]
-ifeval::["{context}" == "installing-gcp-account"]
-:ipi:
-endif::[]
-ifeval::["{context}" == "installing-gcp-user-infra"]
-:upi:
-endif::[]
-ifeval::["{context}" == "installing-gcp-account"]
-:ipiupi:
-endif::[]
-ifeval::["{context}" == "installing-gcp-user-infra"]
-:ipiupi:
-endif::[]
 
 [id="installation-gcp-permissions_{context}"]
-= Required GCP permissions
+= Required GCP roles
 
-When you attach the `Owner` role to the service account that you create, you
-grant that service account all permissions, including those that are required to
-install {product-title}. To deploy an {product-title} cluster, the service
-account requires the following permissions. If you deploy your cluster into an existing VPC, the service account does not require certain networking permissions, which are noted in the following lists:
+When you attach the `Owner` role to the service account that you create, you grant that service account all permissions, including those that are required to install {product-title}. If your organization's security policies require a more restrictive set of permissions, you can create a service account with the following permissions. If you deploy your cluster into an existing virtual private cloud (VPC), the service account does not require certain networking permissions, which are noted in the following lists:
 
 .Required roles for the installation program
 * Compute Admin
 * IAM Security Admin
 * Service Account Admin
```

Comment:
Suggest to move `Service Account Key Admin` after the `Service Account Admin`, so as to order alpha-numerically, thanks!